ssc/Pos/Ssc/Error/Seed.hs

```diff
@@ -1,4 +1,4 @@
--- | Error which can arise during seed calculation.
+-- | Possible failures during SSC seed calculation.
```

Comment:
Shared seed calculation seed calculation :trollface: But okay, it's clearer this way.

Reply:
* `shared seed computation seed calculation`, I guess :)